topics/resolution_algorithms.md

```diff
@@ -1,42 +1,65 @@
 [//]: # (title: Resolution Algorithms)
 
-<include src="lib.md" include-id="outdated_warning"/>
-
-[Routing](Routing_in_Ktor.md) is organized in a tree with a recursive matching system that is capable of handling quite complex rules
-for request processing. The Tree is built with nodes and selectors. The Node contains handlers and interceptors,
-and the selector is attached to an arc which connects another node. If selector matches current routing evaluation context,
-the algorithm goes down to the node associated with that selector.
-
-Routing is built using a DSL in a nested manner:
+[Routing](Routing_in_Ktor.md) is organized in a tree with a recursive matching system that is capable of handling quite
+complex rules for request processing. The Tree is built with nodes that contain selector and optional handler. Selectors
+are used to pick a route based on request. Every selector has quality, usually from 0.0 to 1.0, with one special case
+of "transparent quality" (-1.0). This quality is useful when you need to wrap your subtree in a block, but this block
```

Comment:
This part would need clarifying. It introduces the concept of "block" but not clear what a block is. Also, doesn't really explain what the values of 0 to 1 mean.

Reply:
Updated it to "routing block". I believe the next sentence gives a good example.